### ISSUE TITLE
[WalletConnect Safe app] Allow multiple connections in parallel

### ISSUE THREAD

**Issue by tschubotz** [gnosis/safe-react-apps#80]
# What is this feature about? (1 sentence)

This will allow to have multiple connections in the WalletConnect Safe app open at the same time.

# Why is it needed? What is the value? For whom do we build it?

- When interacting with multiple dapps via WalletConnect at the same time, users would have to close the connection and reconnect every time, when switching between dapps.

# High-level overview of the feature

- Multiple WalletConnect connections can be established at the same time.
- There is a list of open connections.
- Users can close single connections.

# Open questions

- Should we limit the max number of open connections?
- Should we add a "close all connections" button?
- How should the connections be sorted? (Alphabetically by dapp name or by the order they were opened?)

**Comment by mikheevm** [comment]
Also, it might be handly to show which dapp actually triggered the transaction

**Comment by tschubotz** [comment]
There was a user requesting this feature:https://discord.com/channels/477106835862716416/477391201708802058/819935984840867850

**Comment by jdbertron** [comment]
Yeah, that would be nice. Anyone wanting to test a contract on hardhat in conjunction with an existing L1 or L2 vendor contract (like ChainLink VRF)  is facing this problem.
